virttest/remote.py

```diff
@@ -1359,16 +1359,28 @@ def run(self, command, timeout=60, ignore_status=False):
 class VMManager(object):
     """Manage VM on remote host"""
 
+    CMD_TIMEOUT = 240
+
     def __init__(self, params):
         self.remote_host = params.get("server_ip")
         self.remote_user = params.get("server_user")
         self.remote_pwd = params.get("server_pwd")
+        self.vm_ip = params.get("vm_ip")
+        self.vm_pwd = params.get("vm_pwd")
+        self.vm_user = params.get("vm_user", 'root')
```

Comment:
I assume some parameters are mandatory, so do we need do pre-check whether they are valid or not?